### ISSUE TITLE
Meta: Makefile does not properly run emu-algify

### ISSUE THREAD

**Issue by domenic** [whatwg/console#104]
It's hard to do a quick local/remote build; you have to resort to ./deploy.sh --local.

**Comment by domfarolino** [comment]
Yeah honestly in `package.json` I've been using

```
"start": "make remote && emu-algify < index.html > index.tmp.html && rm index.html && mv index.tmp.html index.html"
```

**Comment by domenic** [comment]
Note that a problem with make remote is that it doesn't give you warnings. I guess we should look into surfacing those better somehow.

**Comment by xfq** [comment]
Can this issue be closed now, as #116 has been merged?

**Comment by domenic** [comment]
Ah, yep!
